components/eventing-controller/README.md

```diff
@@ -56,7 +56,7 @@ This section explains how to use the Eventing Controller. It expects the followi
 | `PUBLISHER_LIMITS_MEMORY`         | The memory limits of the Event Publisher Proxy.                                                |
 | **For NATS**                      |                                                                                                |
 | `NATS_URL`                        | The URL for the NATS server.                                                                   |
-| `EVENT_TYPE_PREFIX`               | The Event Type Prefix for the NATS backend.                                                    |
+| `EVENT_TYPE_PREFIX`               | The event type prefix for the NATS backend.                                                    |
```

Comment:
The prefix is used for both NATS and BEB as well.

Reply:
Fixed